File: java/Euler6.java
public class Euler6 {
    // https://projecteuler.net/problem=6

    public static void main(String[] args) {
        int sumSquares = 0;
        double squareSum = 0;

        for (int n = 1; n <= 100; n++) {
            double square = Math.pow(n, 2);
            sumSquares += square;
            squareSum += n;
        }

        squareSum = Math.pow(squareSum, 2);

        System.out.print((int)(squareSum - sumSquares));
    }
}

File: python/euler6.py
# https://projecteuler.net/problem=6

limit = 100  # Find the sum of squares and square of sum of natural numbers up to this number
sum_of_squares = 0
square_of_sum = 0

for i in range(1, limit + 1):
    sum_of_squares += i ** 2
    square_of_sum += i

square_of_sum = square_of_sum ** 2

print(f'The difference between the sum of the squares and the square of the sum of the first {limit} natural numbers is {square_of_sum - sum_of_squares}.')